packages/README.md

```diff
@@ -1,3 +1,82 @@
 # Zarf Packages
 
 This folder contains packages maintained by the [Zarf team](https://github.com/defenseunicorns/zarf/graphs/contributors).  Some of these packages are used by `zarf init` for new cluster initialization.
+
+**Packages**
+- [distros](#distros)
+- [gitea](#gitea)
+- [logging-pgl](#logging-pgl)
+- [zarf-agent](#zarf-agent)
+- [zarf-registry](#zarf-registry)
+
+### Distros
+
+The distros package adds optional capabilities for spinning up and tearing down clusters.  Currently, the following distros are supported:
+
+- [EKS](https://aws.amazon.com/eks/) - Zarf deploys and tears down using the `eksctl` binary under the hood. See how it's done in the EKS package's [`zarf.yaml`](./distros/eks/zarf.yaml) and checkout the [EKS package's config](./distros/eks/eks.yaml) for more information.
+
+- [k3s](https://k3s.io/) - Zarf deploys and tears down using the `k3s` service under the hood. See how it's done in the k3s package's [`zarf.yaml`](./distros/k3s/common/zarf.yaml).
+
+
+#### Usage Examples  
+  
+**EKS**  - Create/Deploy EKS cluster. (requires `eksctl` credentials)
+
+```bash
+zarf package create packages/distros/eks -o build --confirm
+
+zarf package deploy build/zarf-package-distro-eks-multi-x.x.x.tar.zst --components=deploy-eks-cluster --set=CLUSTER_NAME='zarf-nightly-eks-e2e-test',INSTANCE_TYPE='t3.medium' --confirm
+```
+
+See the [nightly-eks test](../.github/workflows/nightly-eks.yml) for another example.
+
+**k3s** - Create/Deploy a k3s cluster.
+
+```bash
+zarf package create packages/distros/k3s -o build --confirm
+
+zarf package deploy build/zarf-init-arm64-vx.x.x.tar.zst --confirm
```

Comment:
```suggestion
# Linux with `systemd` and `root` access only (no `sudo`)
zarf init --components=git-server
```

Since `k3s` is included in the default init package we could just show them this as a simpler way for now.